lib/rspec-puppet/example/application_example_group.rb

```diff
@@ -0,0 +1,19 @@
+module RSpec::Puppet
+  # This module provides support for the application type
+  module ApplicationExampleGroup
```

Comment:
I'm going to leave this for now, I imagine it should be easy to solve whenever, just don't want to add such a refactor right now to this PR if thats okay.